scripts/clean-shrinkwrap.js

```diff
@@ -15,29 +15,258 @@
 
 /**
- * Run an npm command
- * @param {Array} command - list of arguments
- * @returns {ChildProcess}
+ * @summary Get a shrinkwrap dependency object
+ * @function
+ * @private
+ *
+ * @param {Object} shrinkwrap - the shrinkwrap file contents
+ * @param {String[]} shrinkwrapPath - path to shrinkwrap dependency
+ * @returns {Object} shrinkwrap object
+ *
+ * @example
+ * const object = getShrinkwrapDependencyObject(require('./npm-shrinkwrap.json'), [
+ *   'drivelist',
+ *   'lodash'
+ * ]);
+ *
+ * console.log(object.version);
+ * console.log(object.dependencies);
  */
-const npm = (command) => {
-  const npmBinary = os.platform() === 'win32' ? 'npm.cmd' : 'npm';
-  return spawn(npmBinary, command, {
-    cwd: path.join(__dirname, '..'),
-    env: process.env,
-    stdio: [ process.stdin, process.stdout, process.stderr ]
-  });
+const getShrinkwrapDependencyObject = (shrinkwrap, shrinkwrapPath) => {
+  return _.reduce(shrinkwrapPath, (accumulator, dependency) => {
+    return _.get(accumulator, [ 'dependencies', dependency ], {});
+  }, shrinkwrap);
+};
+
+/**
+ * @summary Get a cleaned shrinkwrap dependency object
+ * @function
+ * @private
+ *
+ * @description
+ * This function wraps `getShrinkwrapDependencyObject()` to
+ * omit unnecessary properties such as `from`, or `dependencies`.
+ *
+ * @param {Object} shrinkwrap - the shrinkwrap file contents
+ * @param {String[]} shrinkwrapPath - path to shrinkwrap dependency
+ * @returns {Object} pretty shrinkwrap object
+ *
+ * @example
+ * const object = getPrettyShrinkwrapDependencyObject(require('./npm-shrinkwrap.json'), [
+ *   'drivelist',
+ *   'lodash'
+ * ]);
+ *
+ * console.log(object.name);
+ * console.log(object.path);
+ * console.log(object.version);
+ */
+const getPrettyShrinkwrapDependencyObject = (shrinkwrap, shrinkwrapPath) => {
+  const object = getShrinkwrapDependencyObject(shrinkwrap, shrinkwrapPath);
+
+  if (_.isEmpty(object)) {
+    return null;
+  }
+
+  return {
+    name: _.last(shrinkwrapPath),
+    path: shrinkwrapPath,
+    version: object.version,
+    development: Boolean(object.dev),
+    optional: Boolean(object.optional)
+  };
+};
+
+/**
+ * @summary Get the manifest (package.json) of a shrinkwrap dependency
+ * @function
+ * @private
+ *
+ * @param {String[]} shrinkwrapPath - path to shrinkwrap dependency
+ * @returns {Object} dependency manifest
+ *
+ * @example
+ * const manifest = getShrinkwrapDependencyManifest([ 'bluebird' ]);
+ * console.log(manifest.devDependencies);
+ */
+const getShrinkwrapDependencyManifest = (shrinkwrapPath) => {
+  const manifestPath = _.chain(shrinkwrapPath)
+    .flatMap((dependency) => {
+      return [ 'node_modules', dependency ];
+    })
+    .concat([ 'package.json' ])
+    .reduce((accumulator, file) => {
+      return path.join(accumulator, file);
+    }, '.')
+    .value();
+
+  try {
+
+    // For example
+    // ./node_modules/drivelist/node_modules/lodash/package.json
+    return require(`.${path.sep}${manifestPath}`);
```

Comment:
It would work. Since `manifestPath` is generated using `path.join`, it may include backslashes, so I decided to use `path.sep` just for consistency purposes (so we don't have paths like `./node_modules\package.json`).